src/views/product-downloads-view/helpers.ts

```diff
@@ -31,7 +31,7 @@ export const generateDefaultPackageManagers = (
 		{
 			label: 'Ubuntu/Debian',
 			commands: [
-				`wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor | sudo tee /usr/share/keyrings/hashicorp-archive-keyring.gpg`,
+				`wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor | sudo dd of=/usr/share/keyrings/hashicorp-archive-keyring.gpg`,
```

Comment:
```suggestion
				`wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg`,
```

Hello! I came here also chasing gobbledygook in the install output for Vagrant. I have an alternate fix to suggest though via the [Docker install instructions for linux](https://docs.docker.com/engine/install/ubuntu/). `gpg` supports a `-o` flag so we can just use that directly instead.